sw/device/tests/rv_timer_systick_test.c

```diff
@@ -56,6 +57,37 @@ static status_t test_tick(uint32_t tick_hz) {
   return OK_STATUS();
 }
 
+static status_t test_wrap(uint32_t tick_hz) {
+  LOG_INFO("%s: tick_hz = %u", __func__, tick_hz);
+
+  TRY(dif_rv_timer_irq_set_enabled(
+      &timer, kDifRvTimerIrqTimerExpiredHart0Timer0, kDifToggleEnabled));
+
+  TRY(set_tick(tick_hz));
+
+  uint64_t start_counter = UINT64_MAX - (kWrapTimeMillis * tick_hz) / 1000;
+  TRY(dif_rv_timer_counter_write(&timer, kHart, start_counter));
+  TRY(dif_rv_timer_arm(&timer, kHart, 0, UINT64_MAX));
+
+  TRY(dif_rv_timer_counter_set_enabled(&timer, kHart, kDifToggleEnabled));
+  busy_spin_micros(kWrapTimeMillis * 1000 * 2);
+  TRY(dif_rv_timer_counter_set_enabled(&timer, kHart, kDifToggleDisabled));
+
+  uint64_t counter = UINT64_MAX;
+  TRY(dif_rv_timer_counter_read(&timer, kHart, &counter));
+  // Verify that the counter wrapped.
+  TRY_CHECK(counter < start_counter,
+            "Unexpected elapsed time, expected: %x%x, got: %x%x",
```

Comment:
Nit: Do these need to be `%x%08x` to make sense when joined together?

Reply:
This is the only to print 64 bits variables with the log macros.

Reply:
Good catch. Fixed.